core/src/test/java/io/grpc/internal/CensusModulesTest.java

```diff
@@ -548,11 +550,19 @@ public void traceHeadersPropagateSpanContext() throws Exception {
     Metadata headers = new Metadata();
     callTracer.newClientStreamTracer(headers);
 
+    ArgumentMatcher<StartSpanOptions> startSpanOptionsMatcher =
+        new ArgumentMatcher<StartSpanOptions>() {
+          @Override
+          public boolean matches(Object argument) {
+            return Boolean.TRUE.equals(((StartSpanOptions) argument).getRecordEvents());
+          }
+        };
+
     verify(mockTracingPropagationHandler).toBinaryValue(same(fakeClientSpanContext));
     verifyNoMoreInteractions(mockTracingPropagationHandler);
     verify(mockSpanFactory).startSpan(
         same(fakeClientParentSpan), eq("Sent.package1.service2.method3"),
-        any(StartSpanOptions.class));
+        argThat(startSpanOptionsMatcher));
```

Comment:
May worth changing all `any(StartSpanOptions.class)` (except the stubbing) to this